ui/src/StateContainer.js

```diff
@@ -251,22 +253,21 @@ export class StateContainer extends LitElement {
             @end-session="${this.handleEndSession}"
             @enter-form="${this.handleEnterForm}"
             @exit-form="${this.handleExitForm}"
-            @update-user-info="${this.handleUserInfoUpdate}"
-            @update-collection-state=${this.updateCollectionState}
-            @skip-prompt="${this.handleSkipPrompt}"
-            @end-session="${this.handleEndSession}"
```

Comment:
just double checking--was removing @end-session intentional?

Reply:
it's still there